rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
Maybe they were buggy in `com.android.tools.build:gradle:1.3.1` and got fixed in `2.1.2`. Maybe also the tool version update (from `23.0.1` to `23.0.3`) caused some lint errors.